ONSdigital/census-rm-action-scheduler
# Set lat and long in FWMT action instruction

# Motivation and Context

We need to send mandatory `latitude` and `longitude` attributes to FWMT.

# What has changed



* Updated actionInstructionBuilder to include lat and long
* Added new unit test

# How to test?



* Run `mvn clean install` and ensure all tests pass
* Run acceptance tests from [this related branch](https://github.com/ONSdigital/census-rm-acceptance-tests/compare/send-latitude-longitude-to-fwmt)

# Links



https://trello.com/c/s8K5bM23/837-longitude-latitude-not-being-sent-to-fwmt-3
